### PR TITLE
Add Github templates and scaffolding from CLI but re-worded

### PR DESCRIPTION
**Pull request by dapplion**
While sections like contributing may be outdated due to Mesh re-structuring adding these files is much better than having nothing.

Remembering back to my first contribution, this templates are extremely helpful on the first contact with the codebase

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+(Write what happened. Add screenshots, if applicable.)
+
+## Context
+
+#### Network and buidler.config?
+
+(Which network you are using the app with.)
+
+#### Aragon App project?
```
Comment (sohkai) [inline .github/ISSUE_TEMPLATE/bug.md:51]:
```suggestion
#### Aragon app project?
```

Apparently the official term is "Aragon app" (without the second capitalized a) :).

File: .github/ISSUE_TEMPLATE/question.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+## ❓ Questions and Help
+
+### Please note that this issue tracker is not a help form and this issue will be closed.
+
+Please contact us instead. We have a few channels:
+
+- [Aragon Hack](https://hack.aragon.org/docs)
+- [Aragon Forum](https://forum.aragon.org)
+- [Aragon Chat](https://aragon.chat)
```
Comment (sohkai) [inline .github/ISSUE_TEMPLATE/question.md:14]: Lets use the spectrum link instead: https://spectrum.chat/aragon

File: .github/stale.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# Configuration for https://probot.github.io/apps/stale
```
Comment (sohkai) [inline .github/stale.yml:1]: Do you think this bot will be useful here? I'm wondering if we should avoid using it for now, or use what we have in aragon/aragon (i.e. only enabled for PRs).

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
+# Contributing to buidler-aragon
+
+:tada: Thank you for being interested in contributing to an Aragon Project! :tada:
+
+Feel welcome and read the following sections in order to know how to ask questions and how to work on something.
+
+All members of our community are expected to follow our [Code of Conduct](https://wiki.aragon.org/documentation/Code_of_Conduct/). Please make sure you are welcoming and friendly in all of our spaces.
+
+## Get Rewarded
```
Comment (sohkai) [inline CONTRIBUTING.md:9]: I'd probably just leave this section out for now.

Comment (dapplion) [reply CONTRIBUTING.md]: Fixed with https://github.com/aragon/buidler-aragon/pull/132/commits/36cdee616e35e5ce507b29c6a5511e6807737474

File: .github/stale.yml (hunk 3 above)
Comment (dapplion) [reply .github/stale.yml:1]: Replaced the entire stale.yml with what you have in aragon/aragon. Fixed with https://github.com/aragon/buidler-aragon/pull/132/commits/36cdee616e35e5ce507b29c6a5511e6807737474

File: .github/ISSUE_TEMPLATE/question.md (hunk 2 above)
Comment (dapplion) [reply .github/ISSUE_TEMPLATE/question.md]: Fixed with https://github.com/aragon/buidler-aragon/pull/132/commits/36cdee616e35e5ce507b29c6a5511e6807737474

File: .github/ISSUE_TEMPLATE/bug.md (hunk 1 above)
Comment (dapplion) [reply .github/ISSUE_TEMPLATE/bug.md]: Fixed with https://github.com/aragon/buidler-aragon/pull/132/commits/36cdee616e35e5ce507b29c6a5511e6807737474

**Comment by sohkai** [comment]
I've turned on the stale bot for this repo now as well :).

**Pull request merged**